docs/release-notes.md

```diff
@@ -2,6 +2,33 @@
 title: Release Notes
 ---
 
+## v2.2.1
+
+## New features
+
+- Class B Beaconing support (#1906)
+- AU915 frequency support (#1907)
+- Release workflow (#1942)
+
+## Bugfixes
+
+- Fix EU868 concentrator twin documentation (#1919)
+- Changed verbosity of the HTTP communication in logs (#1916)
+
+## Misc
+
+- From now on the ARM template and the LoRaWan device provisioning CLI will be provided in the Starterkit releases
```

Comment:
Should we mention that this functionality has been removed from the Azure Function? We recommend people to use the cli to provision LoRa related devices.

Reply:
Added follow up to the sentence let me know what you think